Resources/Info.plist

```diff
@@ -20,6 +20,8 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1.0</string>
+	<key>CFBundleShortVersionString</key>
+	<string>${CURRENT_PROJECT_VERSION}</string>
```

Comment:
@erkanyildiz I thought it should be `MARKETING_VERSION`. Isn't it?

Note for other reviewers: The behavior is changed in XCode 11, so we need to check it on older XCode and make sure that CI scripts are not affected (not only by this PR but the behavior change in general)

Reply:
Hi,

Actually, I am not sure about which one should be used: the existing `CURRENT_PROJECT_VERSION` or newly introduced `MARKETING_VERSION`. 

As `CURRENT_PROJECT_VERSION` was already being used (currently 1.4.0) in the project, I just added it to Info.plist, so frameworks in release assets can have release version number too.

But now it seems, `CURRENT_PROJECT_VERSION` is intended to be used for build iteration number, and `MARKETING_VERSION` is for release version number. 
It may be better to handle switching to `MARKETING_VERSION` all over the project, in another PR.

Reply:
Hi there,

Maybe `MARKETING_VERSION` is a good fit for `CFBundleShortVersionString`(Xcode 11) and `CURRENT_PROJECT_VERSION` for `CFBundleVersion`. We need to look at the guidelines and trend.

Reply:
As in build version vs release version.

Reply:
In the end at this point we'll use `CURRENT_PROJECT_VERSION` for both `CFBundleShortVersionString` and `CFBundleVersion`. We'll update `CFBundleShortVersionString` to the `MARKETING_VERSION` likely when we'll deprecate Xcode 10.